CUTSCENE/GetCutsceneEndTime.md

```diff
@@ -5,7 +5,7 @@ aliases: ['0x971D7B15BCDBEF99']
 ## _GET_CUTSCENE_END_TIME
 
 ```c
-// 0x011883f41211432a
+// 0x971D7B15BCDBEF99
```

Comment:
I'm not sure what my actions were prior when I first made the mistake so that change was based on Gottfried's response

Reply:
Given that this is already "broken" with the way it currently is because it doesn't pass the pos/rotation, and will always return 0, swapping this *shouldn't* break compatibility with anything.

This also didn't manage to overwrite the original native definition (probably because the native hash is lowercased?) with an `int` return type so it shouldn't break C# codegen, we would just have to make sure that the native that was supposed to be in the enum slot gets named.

![image](https://github.com/user-attachments/assets/8d401b2a-217a-4c61-b194-1d4e3aaf3dc7)